Description: id and hash for Telegram API
	This is based on similar Debian patch, that makes reproducible build.
	If you copying this ebuild to your overlay, or to the gentoo tree,
	please, consider to go to https://my.telegram.org/apps and create
	your unique api_id and api_hash.

diff --git a/Telegram/SourceFiles/config.h b/Telegram/SourceFiles/config.h
--- a/Telegram/SourceFiles/config.h
+++ b/Telegram/SourceFiles/config.h
@@ -264,6 +264,9 @@ w/CVnbwQOw0g5GBwwFV3r0uTTvy44xx8XXxk+Qknu4eBCsmrAFNnAgMBAAE=\n\
 
 #ifdef CUSTOM_API_ID
-#include "../../../TelegramPrivate/custom_api_id.h" // Custom API id and API hash
+#include "../../custom_api_id.h" // Custom API id and API hash
+#elif defined GENTOO_API_ID
+static const int32 ApiId = 17349; // FIXME!!! API id and hash for Official Gentoo package
+static const char *ApiHash = "344583e45741c457fe1862106095a5eb"; // FIXME!!! API hash for Official Gentoo package
 #else
 static const int32 ApiId = 17349;
 static const char *ApiHash = "344583e45741c457fe1862106095a5eb";
